modules/ice/ice.c

```diff
@@ -383,7 +386,16 @@ static bool if_handler(const char *ifname, const struct sa *sa, void *arg)
 	if (!net_af_enabled(baresip_network(), sa_af(sa)))
 		return false;
 
-	lprio = 0;
+	lprio = m->lprio;
```

Comment:
suggest renaming to "local preference", e.g. lpref:

```
priority = (2^24)*(type preference) +
              (2^8)*(local preference) +
              (2^0)*(256 - component ID)
```

Reply:
Thanks renamed